content/docs/learning-guide/tutorials/environments/create-terrain-from-images.md

```diff
@@ -0,0 +1,314 @@
+{{< /tip >}}
+
+
+## Enable the terrain system
+
+First, you must enable the terrain system by adding two level components to the level prefab.
+
+1. In **O3DE Editor**, create a new level or open an existing level.
+
+    {{< tip >}}
+If your level contains the **Atom Default Environment**, delete or hide the Grid, Ground, and Shader Ball entities so that the terrain you build is not visually obstructed.
+    {{< /tip >}}
+
+2. In **Entity Outliner**, select the Level.prefab.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/enable-terrain-select.png" alt="Select the level.prefab." >}}
+
+3. In **Entity Inspector**, choose **Add Component** and add both the **Terrain World** and **Terrain World Renderer** level components to the level.prefab.
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/enable-terrain-components.png" width="600" alt="Add terrain level components." >}}
+
+There are many properties available in the terrain level components that configure the terrain settings for the level. Later in the tutorial, you might need to make some adjustments to these properties.
+
+{{< important >}}
+For this tutorial, you'll set the world size of the heightmap image so that one pixel is equal to one meter. The rendering distances might need to be adjusted to make the entire terrain visible if the heightmap image is larger than the default terrain rendering distance of 4096 meters or the default camera view distance of 1024 meters.
+
+1. In the **Terrain World Renderer** level component, set the **Mesh render distance** to a value that is greater than the resolution of the heightmap, so that the entire terrain will be visible.
+
+2. If the heightmap is larger than 256 x 256, do the following:
+
+    i. Set the **First LOD distance** property to `256`. This sets the terrain's highest level of detail (LOD) display to a distance of 256 meters from the camera.
+
+    ii. Set the **CLOD Distance** property to `32`. This sets the blend distance between LODs to 32 meters. Larger values make the LOD transitions smoother, but the value should be less than 1/4 the **First LOD distance** value to ensure that there is enough distance to blend each LOD.
+
+3. In the Camera entity, in the **Camera** component, set the **Far clip distance** property to a value that is greater than the resolution of the heightmap.
+
+4. In the Camera entity, in the **Camera** component, choose **Be this camera** to view the level through the camera entity.
+
+5. In the viewport camera settings in the upper-right of the viewport {{< icon "camera.svg" >}} set the **Camera Speed** to an appropriate value for the size of the terrain. `100.0` is a good reference point for large terrains.
```

Comment:
100.0 is just a reference point. It might be too fast or too slow depending on the user/terrain size, but it a good place to start to determine what is a comfortable camera speed with large terrains.